### PR TITLE
Cortex XDR endpoints health check

### PR DESCRIPTION
**Pull request by EliorKedar**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/31417

## Description
This playbook will query disconnected/lost Cortex XDR endpoints with a provided last seen time range playbook input.
The playbook will also generate a CSV report, including a detailed list of the unhealthy endpoints, and send the report to the provided email addresses.
This playbook also includes an incident type with a dedicated layout to visualize the collected data.

## Screenshots
Cortex_XDR_disconnected_endpoints

## Minimum version of Demisto
- [ ] 5.0.0
- [x] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@michalgold Doc review completed.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml [hunk 1]
```diff
@@ -0,0 +1,867 @@
...
+    skipunavailable: false
+    quietmode: 0
+  "5":
+    id: "5"
+    taskid: 2383bf6b-0fb1-49cf-8993-2063413aa608
+    type: condition
+    task:
+      id: 2383bf6b-0fb1-49cf-8993-2063413aa608
+      version: -1
+      name: Was an disconnected endpoint found?
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml:234]: @ShirleyDenkberg should this be "an" or "a"?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml [hunk 2]
```diff
@@ -0,0 +1,867 @@
...
+              simple: PaloAltoNetworksXDR.Endpoint
+            iscontext: true
+      - - operator: isEqualString
+          left:
+            value:
+              simple: PaloAltoNetworksXDR.Endpoint.endpoint_status
+            iscontext: true
+          right:
+            value:
+              simple: LOST
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml:260]: Is LOST the only relevant status here?

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml [hunk 3]
```diff
@@ -0,0 +1,867 @@
...
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: PaloAltoNetworksXDR.Endpoint.endpoint_status
+                iscontext: true
+              right:
+                value:
+                  simple: LOST
+      fileName:
+        simple: Disconnected XDR endpoints.csv
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml:133]: Consider adding a date to the file name to make it easier for the user.

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml [hunk 4]
```diff
@@ -0,0 +1,867 @@
+id: Cortex XDR disconnected endpoints
+version: -1
+name: Cortex XDR disconnected endpoints
+description: |
+  A Job to periodically query disconnected Cortex XDR endpoints with a provided last seen time range playbook input.
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml:5]: Provide a link for setting up a job documentation

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints_README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+A Job to periodically query disconnected Cortex XDR endpoints with a provided last seen time range playbook input.
```
Comment (David-BMS) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints_README.md:1]: Provide a link for job documentation.

File: Packs/CortexXDR/IncidentTypes/incidenttype-Cortex_XDR_Disconnected_endpoints.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    "autorun": false,
+    "color": "#64B5F6",
+    "days": 0,
+    "daysR": 0,
+    "default": false,
+    "disabled": false,
+    "hours": 0,
+    "hoursR": 0,
+    "id": "Cortex XDR Disconnected endpoints",
+    "layout": "294c4eca-caf4-429d-8277-56676b7bee9d",
```
Comment (David-BMS) [inline Packs/CortexXDR/IncidentTypes/incidenttype-Cortex_XDR_Disconnected_endpoints.json:11]: fix id.

File: Packs/CortexXDR/Layouts/layoutscontainer-Cortex_XDR_Disconnected_endpoints.json [hunk 7]
```diff
@@ -0,0 +1,346 @@
...
+            },
+            {
+                "id": "canvas",
+                "name": "Canvas",
+                "type": "canvas"
+            }
+        ]
+    },
+    "group": "incident",
+    "id": "294c4eca-caf4-429d-8277-56676b7bee9d",
```
Comment (David-BMS) [inline Packs/CortexXDR/Layouts/layoutscontainer-Cortex_XDR_Disconnected_endpoints.json:340]: fix id.

**Review by David-BMS — CHANGES_REQUESTED** [review]
@EliorKedar I got two emails
image
The one from EWS also contained an extra file
image

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml (hunk 2 above)
Comment (EliorKedar) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_disconnected_endpoints.yml:260]: yes

**Comment by EliorKedar** [comment]
> @EliorKedar I got two emails
> image
> The one from EWS also contained an extra file
> image

You getting 2 emails because both Gmail and EWS are enabled.
The EWS extra file and bad formatting is due to  a bug in exchange version 2016

**Review by David-BMS — APPROVED** [review]

**Pull request merged**